tests/utils/test_db.py

```diff
@@ -0,0 +1,175 @@
+from unittest.mock import patch
+
+from page_analyzer.utils.db import (
+    create_check,
+    create_url,
+    get_url_by_name,
+    get_url_checks,
+    get_url_data,
+    get_urls,
+)
+
+
+def test_create_url(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    # when
+    url_id = create_url(url_name)
+
+    # then
+    created_url = get_url_data(url_id)
+    assert created_url["name"] == url_name
+    assert url_id == created_url["id"]
+
+
+def test_get_url_data(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    create_url(url_name)
+
+    # when
+    url_data_result = get_url_data(1)
+
+    # then
+    assert url_data_result["name"] == url_name
+
+
+def test_get_url_data_none(test_database: None):
+    # given
+    url_id = -1
+
+    # when
+    url_data_result = get_url_data(url_id)
+
+    # then
+    assert url_data_result is None
+
+
+def test_get_url_by_name(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    create_url(url_name)
+
+    # when
+    url_id = get_url_by_name(url_name)
+
+    # then
+    assert url_id == 1
```

Comment:
wdyt about this variant?
```suggestion
    # given
    url_name = "https://example.com"
    url_id = create_url(url_name)

    # when
    found_url_id = get_url_by_name(url_name)

    # then
    assert found_url_id == url_id
```